JakinChan200/CS180_Roald
# #17 #18 Add Search and Fix Up CSV

## Overview
Adds search fields per country page
Adds CSV parsing to endpoints
Adjusts CSV parsing to allow array of regions or one region

To do:
Investigate KR page not rendering
Investigate country swapping further

## Materials
- [Issue] #17 #18 


## Pictures
Provide a picture or video of your changes if applicable.

## Manual Testing Guide
- Navigate to a region page
- Note that clicking a category pulls new results
- Note that different countries have different results

Pull requests should receive approval before merge.
